### DISCUSSION TITLE
After auto tracking

### DISCUSSION THREAD

**Discussion by RionIwasaki** [WeirongChen/DeepEdge#4] [Q&A] [answered]
Curious about what we should do after auto-tracking. 
If we only care about a certain sequence of frames (e.g., the interval corresponding to a certain sequence of segments such as some CVCs), I guess we can just go back and manually correct some contours whenever we are not happy about them. 
However, if we want a lot of frames, like the whole utterances (or the entire video), what kind of options do we have? It doesn't seem very efficient to go over all frames and manually correct unsatisfactory contours, but I'm pretty sure we will see some very wildly contoured frames if we auto-track the entire video. I suppose that's where "Run Diagnostics" comes into play to save us, so I would love to know how to use it effectively.

**Comment by WeirongChen** [comment, marked as answer]
Thank you Rion for those questions!

You can let DeepEdge track through all frames in the current video file (or a list of multiple files) overnight unattended. It could take several hours depending on your computer. 
Then use "Run Diagnostics" to filter out potentially bad contours. Play with the parameters of diagnostics to reach the best filtering  criteria. 
See manual for diagnostics at:
https://github.com/WeirongChen/DeepEdge/blob/main/Manual/manual_Diagnostics.pdf
